pxtblocks/builtins/misc.ts

```diff
@@ -8,6 +8,12 @@ export function initOnStart() {
     const onStartDef = pxt.blocks.getBlockDefinition(ts.pxtc.ON_START_TYPE);
     Blockly.Blocks[ts.pxtc.ON_START_TYPE] = {
         init: function () {
+            let colorOverride = pxt.appTarget.runtime?.onStartColor;
```

Comment:
So we special-case the onStart block? Why?

Reply:
on-start is kind of a special builtin block because we give targets a lot of control over where it shows up. For example, in micro:bit it's in the basic category but everywhere else it's in loops. iirc we added this option for cue because they had an on-start that doesn't show up in any category, it was just always on the workspace and couldn't be deleted.